gunagala/imager.py

```diff
@@ -283,6 +284,26 @@ def __init__(self, optic, camera, filters, psf, sky, num_imagers=1, num_per_comp
                 # Not a callable, should be a Simple sky model which just returns AB magnitudes per square arcsecond
                 self.sky_rate[filter_name] = self.SB_to_rate(sb, filter_name)
 
+
+    def set_WCS_centre(self, centre, *args, **kwargs):
+        """
+        Set the WCS CRVALs of the Imager instance to centre.
+
+        Parameters
+        ----------
+        centre : astropy.coordinates.SkyCoord or str
+            Sky coordinates of the image centre. Must be either a SkyCoord object or convertible
+            to one by the constructor of SkyCoord.
+        """
```

Comment:
Should probably note in there that additional keyword arguments will be passed to the `SkyCoord` constructor along with `centre`, and give an example (e.g. `units='deg'`)